lib/round.rb

```diff
@@ -0,0 +1,89 @@
+class Round
+	attr_reader :deck, :turns
+
+	def initialize(deck)
+		@deck = deck
+		@turns = []
+	end
+
+	def current_card
+		current_card = @turns.length
+		@deck.cards[current_card]
+	end
+
+	def take_turn(guess)
+		turn = Turn.new(guess, current_card)
+		@turns << turn
+		return turn
+	end
+
+	def number_correct
+		number_correct = 0
+		@turns.each do |turn|
+			if turn.correct?
+				number_correct += 1
+			end
+		end
+		number_correct
+	end
+
+	def number_correct_by_category(category)
+		correct_in_category = 0
+		@turns.each do |turn|
+			if turn.card.category == category
+				if turn.correct?
+					correct_in_category += 1
+				end
+			end
+		end
+		correct_in_category
+	end
+
+	def percent_correct
+		(number_correct.to_f / @turns.length) * 100
+	end
+
+	def percent_correct_by_category(category)
+		total_in_category = 0
+		@turns.each do |turn|
+			if turn.card.category == category
+				total_in_category += 1
+			end
+		end
+		(number_correct_by_category(category).to_f / total_in_category) * 100
```

Comment:
I like that you are referencing your previous method, number_correct_by_category to help you out here.